### PR TITLE
Replace `ophan-tracker-js` with `@guardian/ophan-tracker-js`

### PR DESCRIPTION
**Pull request by arelra**
## What does this change?

Replace `ophan-tracker-js` with `@guardian/ophan-tracker-js`

## Why?

`ophan-tracker-js` is deprecated and we should replace with `@guardian/ophan-tracker-js`.

v2.0.4 will also bring in this fix to how Ophan constructs its requests: https://github.com/guardian/ophan/pull/5624

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -105,10 +105,10 @@
 	},
 	"dependencies": {
 		"@changesets/cli": "^2.26.2",
+		"@guardian/ophan-tracker-js": "2.0.4",
```
Comment (domlander) [inline package.json:108]: The carat has been removed. Was this change intentional?

Comment (arelra) [reply package.json:108]: Yes I think its good to fix the versions so we have predictable builds. Also makes it easier to keep deps aligned with our consumers. DCR has switched to fixed versions.

**Review by domlander — APPROVED** [review]
